indra/llwindow/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ include(LLImage)
 include(LLWindow)
 include(UI)
 include(ViewerMiscLibs)
+include(FetchContent)
```

Comment:
I used to FetchContent git repositories rather than having to 3P everything. The include though indeed is not needed here (I'll keep using that, but I acknowledge this is not the LL/autobuild way).

I removed that line